zerver/lib/integrations.py

```diff
@@ -510,7 +510,8 @@ def __init__(self, name: str, *args: Any, **kwargs: Any) -> None:
     ),
     WebhookIntegration("insping", ["monitoring"]),
     WebhookIntegration("intercom", ["customer-support"]),
-    WebhookIntegration("jira", ["project-management"]),
+    # Avoid collision with jira-plugin's doc "jira/doc.md".
+    WebhookIntegration("jira", ["project-management"], doc="jira/jira-doc.md"),
```

Comment:
We might be able to come up with a better strategy for how to disambiguate this, but I don't think it's worth spending more attention on now.